Publish blog on provider application tips (#2027)
DFE-Digital/get-into-teaching-app: app/views/content/blog/application-tips-from-a-teacher-training-provider.md

## Choosing the right course for you

Whichever route into teaching you choose, make sure it’s the right one for you. As well as ensuring you meet the [minimum requirements to apply](/steps-to-become-a-teacher#step-2), you should be certain you are applying for the age group and subject that suits you. Also, take geographical location into account when applying as you don’t want to spend hours travelling, particularly if you are relying on public transport. [See more tips on choosing the right course provider for you.](/blog/choosing-the-right-teacher-training-course-provider)

## Work experience

It’s not an insurmountable obstacle if you haven’t had any experience of working with children. You can instead highlight any employment where you believe you have gained transferable skills which could be used in a classroom, such as communication, organisation and time management. You might have organised seminars or delivered training sessions, or you might have music qualifications that could be used in a primary setting or be an amateur astronomer that can offer an enrichment class.

## School experience

> School experience is not compulsory, but it may be a good idea to try to visit a school if only to confirm that you are making the right choice before you submit your application. If you are having difficulty finding a school to visit, you might find that schools offering School Direct courses are more welcoming to an enquiry. Due to the pandemic, some schools are not currently accepting non-essential visitors, so use the [Department for Education’s School Experience Portal](https://schoolexperience.education.gov.uk/) to find details of upcoming opportunities in your area.

Reviewer: ```suggestion
School experience is not compulsory, but it may be a good idea to try to visit a school if only to confirm that you are making the right choice before you submit your application. If you are having difficulty finding a school to visit, you might find that schools offering School Direct courses are more welcoming to an enquiry. Due to the pandemic, some schools are not currently accepting non-essential visitors, so use the [Department for Education’s Get School Experience service](https://schoolexperience.education.gov.uk/) to find details of upcoming opportunities in your area.
```

You may wish to demonstrate your awareness of the hard work and commitment required to undertake teaching as a career but also demonstrate how excited you are to start your journey.

I can’t emphasise enough how important it is to avoid spelling mistakes. Make sure to run it through a spell check before copying it into your application, as demonstrating good communication skills is vital. Read more [tips on your personal statement.](/tips-on-applying-for-teacher-training#your-personal-statement)

## References

And finally, we turn to references. Who you pick is really important. You will need two references. One could be a university or academic tutor and the second could be from a school or somewhere you have worked in a voluntary capacity, or it could be someone from your current employment.

Whoever you choose, you should contact your references before you start writing your application so that they are aware that they will be contacted and that a swift response is important. If your references are not ready this can hold up your application. Read more about [choosing your referees.](/tips-on-applying-for-teacher-training#choose-your-referees)

## Responses

> The good news is that as soon as you press the Submit button on your application then it will appear in your chosen providers’ inboxes. Please do not expect an instant response, particularly if you are applying for School Direct places, as the lead school who handles the application process on behalf of a partnership may have to contact your chosen school and wait for them to consider your application and set up an interview. This can take some time to organise, particularly when some schools have two-week timetables.

Reviewer: ```suggestion
The good news is that as soon as you press the Submit button on your application then it will appear in your chosen providers’ inbox. Please do not expect an instant response, particularly if you are applying for School Direct places, as the lead school who handles the application process on behalf of a partnership may have to contact your chosen school and wait for them to consider your application and set up an interview. This can take some time to organise, particularly when some schools have two-week timetables.
```

Reviewer: Also 'Submit' being capitalised doesn't feel quite right maybe
```suggestion
The good news is that as soon as you submit your application then it will appear in your chosen providers’ inboxes. Please do not expect an instant response, particularly if you are applying for School Direct places, as the lead school who handles the application process on behalf of a partnership may have to contact your chosen school and wait for them to consider your application and set up an interview. This can take some time to organise, particularly when some schools have two-week timetables.
```
